.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,19 @@
-## SP 800-63-3 Public Preview Has Closed!
+### All Fields Are Required
 
-Please check back in the coming weeks for the launch of our next public comment phase, where collection of issues and comments will resume. We may have even resolved the issue you were planning on submitting by then - and if not - please submit when we commence.  Until then, all issues received will be automatically closed.
+**Organization**:
 
-Thank you for your interest in supporting and improving NIST SP 800-63-3.
+**Organization Name (self, if individual):**
+ 
+**Type**:
```

Comment:
I'm not clear on what "Type" is. Organization type? Comment type (technical, editorial, etc.)?